### ISSUE TITLE
Create a new logo for the Aragon Cooperative

### ISSUE THREAD

**Issue by lkngtn** [aragoncoop/bounties#1]

**Comment by sepu85** [comment]
yep!! I'd personally like to see the starling bird as our Coop mascote/logo

https://www.youtube.com/watch?v=dedVszDI9aE

**Comment by sepu85** [comment]
linking an issue from the LandingPageRepo --> https://github.com/aragoncoop/Landing-Page/issues/1
